docs/api/web-contents.md

```diff
@@ -218,6 +218,17 @@ When in-page navigation happens, the page URL changes but does not cause
 navigation outside of the page. Examples of this occurring are when anchor links
 are clicked or when the DOM `hashchange` event is triggered.
 
+#### Event: 'will-create-webview'
```

Comment:
We already have `will-attach-webview`, could that be used instead?

https://electron.atom.io/docs/api/web-contents/#event-will-attach-webview